package.json

```
@@ -8,7 +8,7 @@
     "_build": "npm run _hugo -- -e dev --buildDrafts --buildFuture --baseURL \"${DEPLOY_PRIME_URL:-http://localhost}\"",
     "__check:format": "./scripts/npx-helper.sh prettier",
     "_check:format:any": "npm run __check:format -- --check --ignore-path ''",
-    "_check:format:ja+zh": "npm run _check:format:nowrap -- content/ja content/zh",
+    "_check:format:ja+zh": "npm run _check:format:nowrap -- content/ja content/uk content/zh",
```

Shouldn't the command also be updated?

```suggestion
    "_check:format:ja+uk+zh": "npm run _check:format:nowrap -- content/ja content/uk content/zh",
```

Most likely this isn't the only reference, there are definitely other places where `npm run _check:format:ja+zh` must be used

---

Good catch. I'll change the script name in a next round so that we don't have to keep on adding language codes :).